openbb_platform/providers/oecd/openbb_oecd/models/long_term_interest_rate.py

```diff
@@ -138,24 +90,30 @@ def extract_data(
     ) -> Dict:
         """Return the raw data from the OECD endpoint."""
         frequency = query.frequency[0].upper()
-        country = "" if query.country == "all" else country_to_code[query.country]
+        country = "" if query.country == "all" else COUNTRY_TO_CODE_IR[query.country]
         url = "https://sdmx.oecd.org/public/rest/data/OECD.SDD.STES,DSD_KEI@DF_KEI,4.0/..IRLT...."
         data = helpers.get_possibly_cached_data(
             url, function="economy_long_term_interest_rate"
         )
-        query = f"FREQ=='{frequency}'"
-        query = query + f" & REF_AREA=='{country}'" if country else query
+        url_query = f"FREQ=='{frequency}'"
+        url_query = url_query + f" & REF_AREA=='{country}'" if country else url_query
         # Filter down
```

Comment:
So here, we would normally want these sections in the "transform_data", correct?